products/views.py

```diff
@@ -43,6 +43,25 @@ def get(self, request):
 
         return JsonResponse({'products':products_lst}, status=200)
 
+class ProductView(View):
+    def get(self, request, product_id):
+        if not Product.objects.filter(id=product_id).exists():
+            return JsonResponse({"MESSAGE":"INVALID_ID"}, status=404)
+
+        product       = Product.objects.get(id=product_id)
+        product_data = {
+            'id'           : product.id,
+            'name'         : product.name,
+            'thumbnail'    : product.thumbnail_image_url,
+            'options'      : [{
+                'size'  : option.size,
+                'price' : option.price
+                                } for option in product.option_set.all()],
+            'tags'         : [tag.name for tag in product.tags.all()],
```

Comment:
```suggestion
            'options'      : [{
                'size'  : option.size,
                'price' : option.price
             } for option in product.option_set.all()],
            'tags'         : [tag.name for tag in product.tags.all()],
```
다음과 같이 수정하는 것이 가독성 측면에서 더 좋을 것 같습니다.